financialaid/admin.py

```diff
@@ -0,0 +1,57 @@
+"""
+Admin views for Financial Aid app
+"""
+
+from django.contrib import admin
+
+from financialaid.models import (
+    Coupons,
+    Documents,
+    FinancialAid,
+    FinancialAidAudit,
+    TierProgram,
+    Tiers,
+)
+
+
+class CouponsAdmin(admin.ModelAdmin):
+    """Admin for Coupons"""
+    model = Coupons
+
+
+class DocumentsAdmin(admin.ModelAdmin):
+    """Admin for Documents"""
+    model = Documents
+
+
+class FinancialAidAdmin(admin.ModelAdmin):
```

Comment:
There is a possible data privacy question as to whether or not we should expose this model in the django admin.  We haven't finalized auditing requirements yet, but it is possible we will have to track any time someone views the name and income of a person.  If this is exposed through the admin, someone will be able to view this information without us being able to track it